src/components/dashboard/ClaimButton.js

```diff
@@ -109,27 +109,24 @@ const ButtonContent = ({ isCitizen, entitlement, nextClaim, styles, showLabelOnl
   return <ButtonCountdown styles={styles} nextClaim={nextClaim} />
 }
 
-const ClaimButton = ({ isCitizen, entitlement, nextClaim, onPress, styles, style, showLabelOnly, isInQueue }) => {
-  const _onPress = useOnPress(onPress)
```

Comment:
@mauriguz you didnt fix the usage of useOnPress but you removed it.
i dont understand

Reply:
@sirpy correct. claimbutton doesn't always have an onPress handler (specifically when you already claimed and the counter is displayed), that crashes the useOnPress as it expects a callback.
an alternative solution is to add inside of the hook a check for existence of callback, but that would be more like a workaround for specific cases. from my point of view useOnPress logic should always receive a handler.
that's the reason why I removed it instead of adding a conditional to the hook.

cc @serdiukov-o-nordwhale

Reply:
why not pass a default noop? ie onPress || () => {}?

Reply:
@mauriguz yes, you could:

```javascript
import { noop } from 'lodash'
```

then, in component declaration:
```javascript
const ClaimButton = ({ isCitizen, entitlement, nextClaim, onPress = noop, 
```

Reply:
fixed. added noop in most native component using useOnPress